### ISSUE TITLE
Refined storage grids(wireless, crafting, grid) have stopped opening up the GUI

### ISSUE THREAD

**Issue by illmatix** [NillerMedDild/Enigmatica6#724]
**Modpack version**
Enigmatica6 0.3.2
forge 35.1.13

**Issue**
TLDR: Refined storage grids(wireless, crafting, grid) have stopped responding when trying to use them. 

System was working correctly about an hour before. I was in the process of setting up a new network just to handle some plant growing using the growing pots. In this I had the growing pot on top of the storage drawer and an importer below the storage drawer. This was functional when I had exited the game though not exactly great. Items were entering the new network and storing on the disks. 

Both networks were working prior to exiting the game. The area was chunk loaded and claimed. When loading back into the game I had  tried moving the controller, connecting it directly without touching other RS blocks besides a cable, and even spawned in a new controller to try that. but in each case the RS system wouldn't respond to my interactions.

**Logs**
Here is a debug from running /debug and profiling trying to use it.
[debug.zip](https://github.com/NillerMedDild/Enigmatica6/files/5761108/debug.zip)

Latest game log
[latest.log](https://github.com/NillerMedDild/Enigmatica6/files/5761111/latest.log)

**Comment by Darkere** [comment]
Have you tried a client restart? There is a bug around that breaks GUIs on launch sometimes

**Comment by NillerMedDild** [comment]
This sounds very much like #667
